Feat/category

Shin-Yu-1 opened the pull request:
## 개요
 - eslint, prettier 설정 변경
 - html 구조 변경
 - reset.css 추가
 - global.css에서 main.css 변경
 - main.css에서 layaout 작업함 (수정 필요)

## PR 유형
어떤 변경 사항이 있나요?

- [ ] 새로운 기능 추가
- [ ] 버그 수정
- [x] CSS 등 사용자 UI 디자인 변경
- [ ] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [x] 코드 리팩토링
- [x] 주석 추가 및 수정
- [ ] 문서 수정
- [ ] 테스트 추가, 테스트 리팩토링
- [ ] 빌드 부분 혹은 패키지 매니저 수정
- [x] 파일 혹은 폴더명 수정
- [x] 파일 혹은 폴더 삭제

## 💬 공유사항 to 리뷰어
html 태그 및 css를 임의로 수정했습니다.
레이아웃은 작성된 태그 기준으로 잡혀있습니다.
background color를 사용하여 html 실행 시 분리되어 보일 수 있도록 표시했습니다.
색상보고 놀라지 마시고 css를 수정해주세요.

추후 태그와 레이아웃은 알맞게 수정이 더 필요할 것 같습니다.


## PR Checklist
PR이 다음 요구 사항을 충족하는지 확인하세요.

- [x] 커밋 메시지 컨벤션에 맞게 작성했습니다. 
- [x] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

Shin-Yu-1 committed:
fix: 후행콤마 충돌 발생
prettierrc에서 trailingComma을 es5 에서 none으로 변경

Shin-Yu-1 committed:
add: layout, style, category file
- html 태그 변경
- reset css, main css 추가
- category class 추가

Shin-Yu-1 committed:
fix: lint error로 인한 설정 추가
- Uncaught SyntaxError: import declarations may only appear at top level of a module 발생함
- 위 에러 해결을 위해 rules에 import/extensions 추가

Shin-Yu-1 committed:
add: index.html에 script 추가

Shin-Yu-1 committed:
refactor: remove global.css

Shin-Yu-1 committed:
refactor: add entry point

Shin-Yu-1 committed:
add: Roboto font 추가

Jammanb0 reviewed (approved):
확인했습니다. 고생하셨습니다 👍

- 실수로 PR 유형의 체크 박스를 눌러서 원상복귀 시켜놨습니다..

yuj118 reviewed (approved):
확인했습니다 ㅎㅎ 수고하셨어요!

Shin-Yu-1 committed:
fix: Prefer default export on a file with single export
(ESLint rule: import/prefer-default-export) 발생
- export 하나만 있는 파일에서 default export를 사용하도록 권장하는 린트로 인하여 발생함.
- eslint rules에 "import/prefer-default-export": "off" 추가하여 하나만 있는 export도 가능하도록 수정함

Shin-Yu-1 committed:
feat: util 함수 추가
- html element 추가하는 중복 코드 발생
- util.js에 html element 원하는 속성에 맞게 생성될 수 있는 util 함수 추가함.

hyuneeekang commented:
수고 많으셨습니다!

hyuneeekang commented on src/utils/util.js:
😮👍 ..

zi-eunn reviewed (commented):
수고하셨어요!!

hyew0 commented:
확인했습니다. 수고하셨어요!
